main.go

```diff
@@ -286,15 +295,18 @@ func main() {
 		SheetGroups: map[Environment]SheetGroup{
 			dev: {
 				AutomatedSheetName: "PasswordManager-DEV",
-				SheetName:          os.Getenv("MACFINSHEETNAMEDEV"),
+				PortalSheetName:    os.Getenv("PORTALSHEETNAMEDEV"),
+				TestingSheetNames:  strings.Split(os.Getenv("DEVPORTALTESTINGSHEETNAMES"), ","),
```

Comment:
How about  making a utility function that returns an empty list if the string is empty, or the `Split` result otherwise? Then you don't need to add more logic later to deal with empty strings

Reply:
I added a function `getTestingSheets` and removed logic in `updateTestingSheets` that checked for empty string.